maintainer/install_hole.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+# Install HOLE from http://www.smartsci.uk/hole/
+#
+# arguments
+#    OSNAME : linux | darwin
+#    PREFIX : "path/to/installdir"
+#
+# PREFIX can be relative to HOME or an absolute path; it will be created
+# and HOLE unpacked under PREFIX (the tar file contains "hole2/...")
+
+set -o errexit -o nounset
+
+SCRIPT=$(basename $0)
+
+DOWNLOAD_URL_LINUX='https://www.dropbox.com/s/jukpwlohhi20r17/hole2-NotForProfit-2.2.004-Linux-x86_64.tar.gz?dl=1'
+TARFILE_LINUX='hole2-NotForProfit-2.2.004-Linux-x86_64.tar.gz'
+
+DOWNLOAD_URL_DARWIN='https://www.dropbox.com/s/5mzrsyp48i32je4/hole2-NotForProfit-2.2.004-Darwin-i386.tar.gz?dl=1'
+TARFILE_DARWIN=hole2-NotForProfit-2.2.004-Darwin-i386.tar.gz
+
+# path to dir with executables in the current HOLE distribution
+HOLE_EXE_DIR=hole2/exe
+
+function die () {
+    local msg="$1" err=$2
+    echo "[${SCRIPT}] ERROR: $msg [$err]"
+    exit $err
+}
+
+function is_native_executable () {
+    local filename="$1" OSNAME="$2"
+    file "${filename}" | grep -qi ${OFORMAT}
+    return $?
+}
+
+OSNAME="$1"
+case "$OSNAME" in
+    Linux|linux)
+	OSNAME=Linux
+	OFORMAT=Linux
+	DOWNLOAD_URL="${DOWNLOAD_URL_LINUX}"
+	TARFILE=${TARFILE_LINUX}
+	;;
+    OSX|osx|Darwin|darwin)
+	OSNAME=Darwin
+	OFORMAT="Mach-O"
+	DOWNLOAD_URL="${DOWNLOAD_URL_DARWIN}"
+	TARFILE=${TARFILE_DARWIN}
+	;;
+    *)
+	die "OS '${OSNAME}' not supported." 10;;
+esac
+
+PREFIX="$2"
+test -n "$PREFIX" || die "missing second argument PREFIX (installation directory)" 10
+
+#------------------------------------------------------------
+# start installation
+#------------------------------------------------------------
+
+cd ~
```

Comment:
I think that line is a left-over from an earlier version of the script when I hadn't switched over to using `PREFIX`. I think I can remove it.

In any case, it only has to work on travis...